lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -66,3 +91,58 @@ data ErrMkStdTx
     | ErrInvalidTx
     -- ^ when transaction with 0 amount is tried (not valid in Byron)
     deriving (Eq, Show)
+
+-- | Backend-specific variables used by 'estimateMaxNumberOfInputsBase'.
+data EstimateMaxNumberOfInputsParams = EstimateMaxNumberOfInputsParams
+    { estMeasureTx :: [TxIn] -> [TxOut] -> [TxWitness] -> Int
+        -- ^ Finds the size of a serialized transaction.
+    , estAddressSample :: Address -- ^ Address to use in tx output
```

Comment:
I would have liked to use a plain `estAddressSize :: Int` but the Tx serializer wants to deserialize the addresses it is given...

Reply:
I see :thinking: .. This is only in the bridge I guess but that makes sense.
Though, I record parameter is now superfluous since we have access to `t`. We can construct a valid address for a target network using only a public key, and public keys always have the same size. Hence:

```hs
baseAddr = keyToAddress @t <$> (xpub $ chaff 64) 
```

Reply:
Ah yes, that simplifies things. Fixed.